### PR TITLE
Add alt text to landing page images

### LINKED ISSUE #201: Add alt tags to decorative images on home page
Should be `alt=""`

@batpigandme can you handle please?

### PR DESCRIPTION
**Pull request by batpigandme**
* Fixes #201

### CODE REVIEW

File: layouts/index.html [hunk 1]
```diff
@@ -42,7 +42,7 @@
 
   <div class="band third">
     <div class="bandContent">
-      <div class="hideOnMobile"><img class="bee1" src="images/bee1.jpg"></div>
+      <div class="hideOnMobile"><img class="bee1" src="images/bee1.jpg" alt="bee"></div>
```
Comment (hadley) [inline layouts/index.html]: I think this should definitely be `alt=""` since it's purely decorative

File: layouts/index.html [hunk 2]
```diff
@@ -31,7 +31,7 @@
           See how the tidyverse makes data science faster, easier and more fun with "R for Data Science".
           Read it <a href="http://r4ds.had.co.nz/">online</a>, buy <a href="http://amzn.to/2aHLAQ1">the book</a> or try another <a href="/learn/">resource from the community</a>.
         </div>
-        <a href="http://amzn.to/2aHLAQ1"><img class="bookCover" src="images/cover.png"></a>
+        <a href="http://amzn.to/2aHLAQ1"><img class="bookCover" src="images/cover.png" alt="R for Data Science book cover"></a>
```
Comment (hadley) [inline layouts/index.html]: I think this could be `alt=""` because the image doesn't add anything not already in the text?

**Comment by hadley** [comment]
Can you please also take a pass at the help page?

**Comment by batpigandme** [comment]
Yeah, it's actually in https://github.com/tidyverse/tidyverse.org/pull/204

**Pull request merged**